Wrappers/Python/cil/io/NikonDataReader.py

```diff
@@ -22,66 +22,68 @@
-        mode: str, 'bin' (default) or 'slice'. In bin mode, 'step' number
-                of pixels is binned together, values of resulting binned
-                pixels are calculated as average. 
-                In 'slice' mode 'step' defines standard numpy slicing.
-                Note: in general 
-                output array size in bin mode != output array size in slice mode
+    file_name: str 
+        full path to .xtekct file
         
-        Output
-        ------
+    roi: dict
+        dictionary with roi to load:
+        {'angle': (start, end, step), 
+            'horizontal': (start, end, step), 
+            'vertical': (start, end, step)}
         
-        Acquisition data with corresponding geometry, arranged as ['angle', horizontal'] 
-        if a single slice is loaded and ['vertical, 'angle', horizontal'] 
-        if more than 1 slices are loaded.
-                    
-        '''
+    normalise: bool, default=True
+        normalises loaded projections by detector white level (I_0)
+                        
+    fliplr: bool, default = False,
+        flip projections in the left-right direction (about vertical axis)
+                        
+    mode: str: {'bin', 'slice'}, default='bin'
+        In bin mode, 'step' number of pixels is binned together,
+        values of resulting binned pixels are calculated as average. 
+        In 'slice' mode 'step' defines standard numpy slicing.
+        Note: in general output array size in bin mode != output array size in slice mode
+
+
+    Notes
+    -----
+    `roi` behaviour:
+        Files are stacked along axis_0. axis_1 and axis_2 correspond
+        to row and column dimensions, respectively.
+        
+        Files are stacked in alphabetic order. 
         
-        self.file_name = kwargs.get('file_name', None)
-        self.roi = kwargs.get('roi', {'angle': -1, 'horizontal': -1, 'vertical': -1})
-        self.normalise = kwargs.get('normalise', True)
-        self.mode = kwargs.get('mode', 'bin')
-        self.fliplr = kwargs.get('fliplr', False)
+        To skip projections or to change number of projections to load, 
+        adjust 'angle'. For instance, 'angle': (100, 300)
+        will skip first 100 projections and will load 200 projections.
         
-        if self.file_name is not None:
-            self.set_up(file_name = self.file_name,
-                        roi = self.roi,
-                        normalise = self.normalise,
-                        mode = self.mode,
-                        fliplr = self.fliplr)
+        ``'angle': -1`` is a shortcut to load all elements along axis.
+            
+        ``start`` and ``end`` can be specified as ``None`` which is equivalent
+        to ``start = 0`` and ``end = load everything to the end``, respectively.
+        Start and end also can be negative.
+                
+    '''
+    
+    def __init__(self, file_name = None, roi= {'angle': -1, 'horizontal': -1, 'vertical': -1},
```

Comment:
roi=`None` should be default.

But I don't think this is too important for this PR